assets/notebooks/assembly_report.qmd

```diff
@@ -164,14 +164,82 @@ We can glean a good amount of information about the sequenced species from the k
 #### Ploidy
 Does it show the correct ploidy?
 
-::: {.callout-note title="Interpretation" collapse="true"}
-A Kmer histogram counts the frequency of k-mers in the data. Some parts of the genome are unique, and
-there are expected to be *coverage* times copies of these k-mers in the data. If the organism is diploid,
-the regions unique to a haplotype should have *coverage* times copies of these k-mers, and regions shared
-between both haplotypes should have *coverage* times two copies of these k-mers. This leads to formations
-of peaks in the k-mer histogram. A haploid organism should have one peak, while a diploid should have two peaks.
+
+::: {.callout-note title="what is GenomeScope and how to interpret the GenomeScope plot" collapse="true"}
+#### **GenomeScope Overview**
+
+**GenomeScope** is a computational tool designed to analyze raw sequencing data and provide insights into the genome's characteristics, such as its **size**, **heterozygosity rate**, and **repeat content**. It works by analyzing **k-mer frequency distributions** derived from sequencing reads and fitting a mathematical model to these distributions.
+
+---
+
+##### **How GenomeScope Works**
+
+1. **Input Data**  
+   GenomeScope uses a **k-mer frequency histogram** generated by tools like **Jellyfish**.  
+   - A **k-mer** is a sequence of *k* nucleotides.  
```

Comment:
Maybe format it in a form `- parameter: description` so it's clearer what GenomeScope expects.

Reply:
Actually. Given it's automated, perhaps describing the input is superfluous. Let's limit it to just what it produces, and how to interpret it. We can talk about tool input, etc in workshop material or something instead so it's less of a distraction here when interpreting results.